src/decision/src/VisionDecision.cpp

```diff
@@ -1,15 +1,21 @@
 /*
- * Created By: Gareth Ellis
+ * Created By: Robyn Castro
  * Created On: September 22, 2016
  * Description: The vision decision node, takes in an image from the robot's
  *              camera and produces a recommended twist message
  */
-
 #include <VisionDecision.h>
+#include <tiff.h>
+
+#define PI 3.14159265
+#define null -1
```

Comment:
Since in c++ `NULL` refers to a **null pointer**, this could be confusing. If you really want a "null" value, you should replace all the `int`'s you set to null with `int*`'s instead, and use `NULL`, which is already defined, instead of `null`. Alternatively (and this is probably easier) you can replace "null" with -1 throughout the program. Or something else entirely, I'm just trying to avoid confusion.